content/en/tracing/tracing_without_limits/_index.md

```diff
@@ -0,0 +1,220 @@
+
+A valid query in the search bar displays traces that match your search criteria across **all spans**. The search syntax is the same in the Live Search views as in the other trace views, but here, your query is matched against all of the ingested traces across any span and any tag, and not just the indexed ones.
+
+**Note**: You can select only the `top-level spans of the service` by selecting the checkbox above the trace table. You can use this feature on high traffic applications to reduce the number of spans displayed and view only the entry point spans of the services. Selecting this box only visually filters the spans shown.
+
+You can also filter on attributes that are not defined as facets. For example, to filter on the `customer.id` attribute, there are two options:
+
+- Click on the attribute in the trace panel and add it to the search query `@customer.id:584959`.
+{{< img src="tracing/live_search/livesearch_query2.png" alt="Live Search filter" >}}
+
+
+- Filter on all spans with a `customer.id` attribute by typing "customer.id" in the search query bar: `@customer.id:584959`
+{{< img src="tracing/live_search/livesearch_query1.png" alt="Live Search filter" >}}
+
+### Live Analytics Mode
+
+<div class="alert alert-warning">
+These features are currently in private beta. <a href="https://forms.gle/1FParyX49eNFPDsg9">Fill out the form</a> to request to be added to the beta.
+</div>
+
+{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-2.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
+
+With Live Analytics mode, you can perform analytics on 100% of your ingested traces for the last 15 minutes.  Analytics is used to graph requests or errors corresponding to specified criteria, such as errors for the `ShoppingCart##checkout` service and endpoint, with a cart value of at least `$100`, with the ability to drill directly into traces matching these criteria.
+
+In addition to viewing analytics for requests matching arbitrary queries, Live Analytics can also give a top list of the most impacted customers, availability-zones, or any other tag during an outage or investigation.
+
+With Live Analytics, every tag on every span ingested over the rolling 15 minute window is available for querying.
+
+{{< img src="tracing/live_search/LiveAnalytics.gif" alt="Live Analytics" >}}
+
+**Note:** Exporting to dashboards and monitors is only possible in Historical mode.
+
+### Historical Analytics Mode
+
+{{< img src="tracing/live_search_and_analytics/tracing_without_limits_lifecycle-4.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Journey" >}}
+
+Historical Analytics Mode is accessed in the same way as Live Search mode.  To switch between the two, change the time selector to any period of time greater than 15 minutes.  This will change the data being searched to Indexed Spans, and the data will no longer be a live feed.
```

Comment:
```suggestion
Historical Analytics Mode is accessed in the same way as Live Analytics mode.  To switch from Live to Historical Analytics, change the time selector to any period of time greater than 15 minutes.  This will change the data being searched to indexed spans, and the data will no longer be a live feed.
```